packaging/rhel/syslog-ng.spec

```diff
@@ -34,7 +34,7 @@ BuildRequires: hiredis-devel
 BuildRequires: riemann-c-client-devel
 BuildRequires: python-devel
 BuildRequires: java-devel
-BuildRequires: gradle
```

Comment:
We already decided to use upstream gradle.
Shouldn't we check if that one exists, e.g. with a `gradle -v` call?

Reply:
It would be an improvement to check the gradle version in our configure scripts, to check
if it meets minimum requirements. But I think it would be a mistake to do the same in the spec file.

Also, unfortunately, the version number doesn't seem to be a good enough indicator, the gradle versions did work, but someone (either gradle upstream or debian or ubuntu) br0ke it.

Reply:
We talked about it in person and I accept that we should not check whether gradle is installed locally (with `gradle -v`).